src/commands/junit/README.md

```diff
@@ -30,7 +30,7 @@ datadog-ci junit upload --service my-service --tags key1:value1 --tags key2:valu
 - `--max-concurrency` (default: `20`): number of concurrent uploads to the API.
 - `--dry-run` (default: `false`): it will run the command without the final upload step. All other checks are performed.
 - `--logs` (default: `false`): it will enable collecting logs from the content in the XML reports.
-- `--skip-git-metadata-upload` (default: `false`): it will run the command without uploading git metadata.
+- `--enable-git-metadata-upload` (default: `false`): it will upload git metadata.
```

Comment:
This is a breaking change (if someone was using `--skip-git-metadata-upload` then this will break them. I think we should just change the default value to "true"

Reply:
of course 🤦, good catch

Reply:
this forces us to use something like `--skip-git-metadata-upload=false` or `--skip-git-metadata-upload=0`, which isn't great, but it's better than a breaking change